web/pdf_find_controller.js

```diff
@@ -736,6 +736,7 @@ class PDFFindController {
       state,
       previous,
       matchesCount: this._requestMatchesCount(),
+      rawQuery: this._rawQuery,
```

Comment:
Really sorry, I just looked at this more in-depth and realized that this property is not guaranteed to always be set when `_updateUIState` is called.

The following should be much safer in general:
```suggestion
      rawQuery: this._state ? this._state.query : null,
```
assuming that you also add validation similar to https://searchfox.org/mozilla-central/rev/6cc48251bb97600fdf11a5b4c5f621bfc8606d55/toolkit/components/pdfjs/content/PdfStreamConverter.jsm#554-557, i.e. adding the following in `PdfStreamConverter.jsm` (such that the parameter is optional):
```javascript
const rawQuery = typeof data.rawQuery === "string" ?  data.rawQuery : null;
```

Reply:
I have added:

```js
    // Same for the `rawQuery` property.
    let rawQuery = null;
    if (typeof data.rawQuery === "string") {
      rawQuery = data.rawQuery;
    }
```